client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -1,33 +1,34 @@
 import { makeStyles } from '@material-ui/core/styles';
+import { SignalWifi1BarLockSharp } from '@material-ui/icons';
 
 const useStyles = makeStyles((theme) => ({
   form: {
     width: '100%', // Fix IE 11 issue.
     marginTop: theme.spacing(1),
   },
   label: {
-    fontSize: 19,
-    color: 'rgb(0,0,0,0.4)',
-    paddingLeft: '5px',
+    fontSize: 15,
+    fontWeight: 700,
+    color: 'black',
+    marginBottom: '-0.8rem',
+    marginTop: '0.8rem',
   },
   inputs: {
-    marginTop: '.8rem',
-    height: '2rem',
-    padding: '5px',
+    height: '1.5rem',
   },
   forgot: {
     paddingRight: 10,
     color: '#3a8dff',
   },
   submit: {
-    margin: theme.spacing(3, 2, 2),
-    padding: 10,
+    margin: theme.spacing(3, 2, -7),
+    padding: 0,
     width: 160,
     height: 56,
     borderRadius: theme.shape.borderRadius,
     marginTop: 49,
     fontSize: 16,
-    backgroundColor: '#3a8dff',
+    backgroundColor: 'primary',
```

Comment:
is this actually changing to the string primary? what is primary colour?

Reply:
As with :

what is #primary color here? is this a valid color? did you mean to use the theme's primary colour?

_Originally posted by @bonnieli in https://github.com/hatchways/team-corgi/pull/42#discussion_r652087127_

I have fixed both of the references to our primary colour:

![image](https://user-images.githubusercontent.com/19699016/122114203-0931c080-cdf1-11eb-91c1-c7a4dcddcbee.png)